models/labels/addresses/dex/labels_dex.sql

```diff
@@ -8,6 +9,9 @@
                                 \'["ilemi"]\') }}')
 }}
 
+{# -- Enable once balances work again
+,ref('labels_trader_portfolios')
```

Comment:
@jeff-dude note that I disabled this one because it depends on ethereum balances.

Reply:
how do you want to track this (adding back in future)? backlog task in linear?

Reply:
I've created a task to track it